### PR TITLE
Add command to clear local solr server.

### PR DESCRIPTION
**Pull request by lahoosascoots**
Changes proposed:
- Adds command to clear local solr server.

### CODE REVIEW

File: src/Robo/Commands/Solr/ClearLocalSolrIndexCommand.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+  /**
+   * Clear local solr index.
+   *
+   * @command solr:clear
+   * @description Clears the local solr index.
+   */
+  public function clearSolr() {
+    $this->taskExecStack()
+      ->exec("curl -s http://local.ebsco.com:8983/solr/d8/update --data '<delete><query>*:*</query></delete>' -H 'Content-type:text/xml; charset=utf-8' > /dev/null")
```
Comment (greylabel) [inline src/Robo/Commands/Solr/ClearLocalSolrIndexCommand.php]: `ebsco.com`?

Comment (lahoosascoots) [reply src/Robo/Commands/Solr/ClearLocalSolrIndexCommand.php]: Ah whoops! Good catch. I should pull that from config.

File: src/Robo/Commands/Solr/ClearLocalSolrIndexCommand.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  /**
+   * Clear local solr index.
+   *
+   * @command solr:clear
+   * @description Clears the local solr index.
+   */
+  public function clearSolr() {
+    $this->taskExecStack()
+      ->exec("curl -s {$this->getConfigValue('project.local.uri')}:8983/solr/d8/update --data '<delete><query>*:*</query></delete>' -H 'Content-type:text/xml; charset=utf-8' > /dev/null")
```
Comment (grasmash) [inline src/Robo/Commands/Solr/ClearLocalSolrIndexCommand.php]: Is this always the correct port?

Comment (lahoosascoots) [reply src/Robo/Commands/Solr/ClearLocalSolrIndexCommand.php]: This is the standard solr port which we do not have a config variable for

**Pull request closed without merging**